### ISSUE TITLE
Header - Button «Login» instead of a button «Особистий кабінет» 

### ISSUE THREAD

**Issue by LesiaBezukh** [scholokov/qax-portal-2#105]
Steps To Reproduce:
1. Open the page https://portal.qax-camp.com.ua/
2. Pay attention to the Header 

Actual result: Button «Login»
Expected result: Button «Особистий кабінет» 
<img width="1440" alt="Снимок экрана 2022-09-16 в 10 48 37 AM" src="https://user-images.githubusercontent.com/110661667/190588676-0e80ee35-8c2a-40c8-b709-e2f93daee488.png">
<img width="1440" alt="Снимок экрана 2022-09-16 в 10 50 08 AM" src="https://user-images.githubusercontent.com/110661667/190588709-c7e09bd2-f263-421a-af33-4e177f24a35f.png">
<img width="1440" alt="Снимок экрана 2022-09-16 в 10 52 44 AM" src="https://user-images.githubusercontent.com/110661667/190588768-b61d34e3-0d15-4790-b687-5c68dabc8154.png">

**Comment by DmytroMarkulych** [comment]
@LesiaBezukh Чи змінюється пункт меню Login на Особистий кабінет, після логіну на сайті?

**Comment by LesiaBezukh** [comment]
нет, пункт меню после регистрации вообще отсутствует, мы обсуждали это на митинге

**Comment by YulyaBaday** [comment]
The bug is not relevant, the design of the button has changed.
